randovania/layout/base/base_configuration.py

```diff
@@ -86,6 +88,9 @@ def dangerous_settings(self) -> list[str]:
         if self.first_progression_must_be_local:
             result.append("Requiring first progression to be local causes increased generation failure.")
 
+        if self.logical_pickup_placement is LogicalPickupPlacementConfiguration.ALL:
+            result.append("Placing all pickups logically causes increased generation failure.")
+
```

Comment:
as dark mentioned in an earlier review, please move this to `LogicalPickupPlacementConfiguration.dangerous_settings()`